.github/workflows/rzk.yml

```
@@ -17,9 +21,8 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Check all files
-        uses: rzk-lang/rzk-action@v1
+      - name: Typecheck Rzk files
+        uses: rzk-lang/rzk-action@v1.1.0
```

Now this should work:

```suggestion
        uses: rzk-lang/rzk-action@v1
```